cmd/gpu_plugin/gpu_plugin_test.go

```diff
@@ -314,3 +317,110 @@ func TestScan(t *testing.T) {
 		})
 	}
 }
+
+// Would be nice to combine these with the overall Scan unit tests.
+func createBypathTestFiles(t *testing.T, root, linkFile string, bypathFiles []string) (string, string) {
+	drmPath := path.Join(root, "sys/class/drm/card0")
```

Comment:
Instead of hard-coding this to "card0"  here and in Testbypath(), maybe that could be a variable, so that (in theory) more than single device could be tested?

PS. feel free to ignore, if you don't think this relevant.

Reply:
Can be now set as resolved.